roles/ldap/defaults/main.yml

```diff
@@ -1,8 +1,9 @@
 ---
 # roles/ldap/defaults/main.yml
 project_name: sufia-project
-capistrano_user: deploy
-capistrano_group: deploy
+deploy_user: deploy
+deploy_group: deploy
+config_path: "{{ project_base }}/{{ project_name }}/shared/config"
```

Comment:
and a default for project_base here, otherwise the config_path default won't function